### ISSUE TITLE
Questions about the graph classification results

### ISSUE THREAD

**Issue by 2xingwen** [HongyangGao/gunet#8]
When I train the network. I find its test acc appear very high at the beginning.

Namespace(batch_size=20, data='PROTEINS', dropout=True, extract_features=False, feat_dim=0, fold=1, hidden=128, latent_dim=[32, 32, 32, 1], learning_rate=0.001, max_lv=4, mode='gpu', num_
class=0, num_epochs=100, out_dim=0, printAUC=False, seed=1, sortpooling_k=0.6, test_number=0)
loading data
# classes: 2
# maximum node tag: 3
# train: 1001, # test: 112
k used in SortPooling is: 32
Initializing GUNet
loss: 0.50198 acc: 0.75000:  24%|████████████████████████████▌                                                                                          | 12/50 [00:00<00:01, 33.74batch/s$
/workspace/zwq/GraphPool/gunet/ops.py:13: RuntimeWarning: divide by zero encountered in power
  d_inv_sqrt = np.power(rowsum, -0.5).flatten()
loss: 0.36440 acc: 0.85000: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 50/50 [00:01<00:00, 35.79batch/s$
average training of epoch 0: loss 0.57412 acc 0.69500 auc 0.00000
loss: 0.42157 acc: 0.83333: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6/6 [00:00<00:00, 52.19batch/s$
average test of epoch 0: loss 0.53319 acc 0.73214 auc 0.00000
loss: 0.61608 acc: 0.75000: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 50/50 [00:01<00:00, 36.62batch/s$
average training of epoch 1: loss 0.54999 acc 0.73100 auc 0.00000
loss: 0.35901 acc: 0.91667: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6/6 [00:00<00:00, 52.55batch/s$
average test of epoch 1: loss 0.53982 acc 0.74107 auc 0.00000
loss: 0.46130 acc: 0.75000: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 50/50 [00:01<00:00, 37.91batch/s

Can anyone explain this to me?
